### PR TITLE
[EAM-966] Add Status row, dependent on eam-components change

### PR DESCRIPTION
**Pull request by BoykoBorisov**

### CODE REVIEW

File: src/ui/components/statusrow/StatusRow.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+const iconStyle = {
+    height: "25px",
+    width: "25px"
+}
+
+const STATUSES = [
+    {
+        key: STATUS_KEYS.OUT_OF_SERVICE,
+        shouldRender: (entity, entityType) => entity.outOfService === "true" || entity.outOfService === true,
```
Comment (pedroamarocern) [inline src/ui/components/statusrow/StatusRow.js:20]: Is `entity.outOfService` `true` or `"true"` when enabled? Does it vary depending on the entity? If not, why have both checks?

File: src/ui/components/statusrow/StatusRow.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+]
+
+const StatusRow = (props) => {
+    const generateCells = (entity, entityType) => {
+        return STATUSES.map(status => {
+            if (status.shouldRender(entity, entityType)) {
+                return (
+                    <Tooltip title={status.description}>
```
Comment (pedroamarocern) [inline src/ui/components/statusrow/StatusRow.js:44]: This should display the "full" description (EIS -> Élément Important pour la Sécurité)

File: src/ui/components/statusrow/StatusRow.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                            <div style={{fontSize: "0.75rem"}}>{status.description}</div>
+                        </div>
+                    </Tooltip>
+                )
+            }
+        })
+    }
+
+    const icons = generateCells(props.entity, props.entityType);
+    return icons.length && <div style={{width: "100%", display: "flex", ...props.style}}>{icons}</div>;
```
Comment (pedroamarocern) [inline src/ui/components/statusrow/StatusRow.js:56]: Because icons is an `Array`, a key should be specified in the `Tooltip` component: https://reactjs.org/docs/lists-and-keys.html#keys (you may use `status.key`)

File: src/ui/pages/equipment/asset/AssetGeneral.js [hunk 4]
```diff
@@ -11,7 +12,6 @@ class AssetGeneral extends Component {
 
         return (
             <div style={{width: "100%", marginTop: 0}}>
-
```
Comment (pedroamarocern) [inline src/ui/pages/equipment/asset/AssetGeneral.js:14]: This change is not relevant to the PR. We should avoid having blank line additions/removals in unrelated lines/files.

File: src/ui/pages/part/Part.js [hunk 5]
```diff
@@ -103,7 +104,7 @@ class Part extends Entity {
                         {...commonProps} />
                 ,
                 column: 1,
-                order: 1
+                order: 2
```
Comment (pedroamarocern) [inline src/ui/pages/part/Part.js:107]: Why was the `order` changed here (and all lines below)?

File: src/ui/pages/part/Part.js [hunk 6]
```diff
@@ -16,6 +16,7 @@ import {PartIcon} from 'eam-components/dist/ui/components/icons'
 import EDMSDoclightIframeContainer from "../../components/iframes/EDMSDoclightIframeContainer";
 import {TOOLBARS} from '../../components/AbstractToolbar';
 import EntityRegions from "../../components/entityregions/EntityRegions";
+import StatusRow from '../../components/statusrow/StatusRow'
```
Comment (pedroamarocern) [inline src/ui/pages/part/Part.js:19]: I think this import is unused.

**Review by pedroamarocern — CHANGES_REQUESTED** [review]
Required changes before merging:
* Clarify/fix EIS shouldRender function
* EIS ->Élément Important pour la Sécurité
* `key` in `Tooltip` component
* Clarify `StatusRow` import and order changes in `Part.js`

File: src/ui/components/statusrow/StatusRow.js (hunk 1 above)
Comment (BoykoBorisov) [reply src/ui/components/statusrow/StatusRow.js:20]: It varies, yes.

**Review by pedroamarocern — APPROVED** [review]
Looks good to me. I see now that the `order` property on `Part.js` has a repeated number, well spotted! We can merge it like this for now.
